abdulfatai360/book-finder
# Make Result Appears on Subsequent Search Requests

#### What does this PR do?
Fixes the error that occurs when search requests are made more than once before a browser reload

#### Description of Task to be completed?
When a user performs a first search, results are rendered
But on subsequent search requests, results do not render on the page
So have the result of search requests render on the page without the user having to refresh the browser to restart the application

#### How should this be manually tested?
- Clone this repo: ```https://github.com/abdulfatai360/book-finder.git```
- Navigate to project folder ```book-finder/```
- Open the ```index.html``` file on a browser
- Perform search request multiple times; search result should appear on every attempt
